inc/classes/class_gd.php

```diff
@@ -301,42 +382,7 @@ public function CreateThumb($old_file, $new_file, $max_width, $max_height)
             $this->PutImage($new_file);
             ImageDestroy($imgsrc_old);
         }
-        return true;
-    }
-
-
-    public function Colorize($source_file, $r, $g, $b, $percentage)
-    {
-        if ($source_file) {
-            $this->img = $this->OpenImage($source_file);
-            $this->width = imagesx($this->img);
-            $this->height = imagesy($this->img);
-        }
-
-        // Create Layover
-        $layover = imagecreate($this->width, $this->height);
-        $fill = imagefill($layover, 0, 0, imagecolorallocate($layover, $r, $g, $b));
-
-        // Merge Layover with original image
-        $merge = imagecopymerge($this->img, $layover, 0, 0, 0, 0, $this->width, $this->height, $percentage);
-        imagedestroy($layover);
-    }
-
 
-    public function MergeImages($source_file1, $source_file2, $target_file)
```

Comment:
I would argue that removing this is fine, as:
* imagecopymerge does not support the alpha channel correctly. Merging two pictures with transparency thus would never work as expected.
* Also merging just parts does not work, as coordinates for merge cannot be passed.

I do see the need for a similar function, but that must support the two requirements above

Reply:
👍